tessellator/c/tessellator.dart

```diff
@@ -0,0 +1,149 @@
+// ignore_for_file: camel_case_types
```

Comment:
This file is missing a copyright header. Was this file generated? If so, should it be generated by the build?

Ideally this would also be structured as a proper Dart package (i.e. with a pubspec.yaml, lib/, tests/, examples/, etc.), but that could be follow-on work. Please file an issue.

Another question would be, is this the right place for this file to live? Or should it live directly in the package for the Dart command-line tool that's going to use it? Those could also be answered later.

Reply:
I don't think we could actually use this file as is. The dart source code for package:vector_graphics would need to import it, but assuming this is built in the engine repo the package itself won't know exactly where this file is so it wouldn't be able to use it.

So if we structured it as a package, at least the package could be added as a dependency and the location configured via  pubspec.yaml or blaze or something

Reply:
This was not generated. I can add the header.

One problem right now is that there's no standard way of distributing FFI packages on pub. @dcharkes has shared an example with me that's Flutter based, but I haven't really worked out how we would distribute this. I would like the Dart wrapper to exist here so that it can be updated as needed, but maybe I'm doing the wrong thing and we should just see what `ffigen` produces for the header file...

Reply:
IMO distribution via pub is unimportant right now - the question of how we use this in g3 is the same as everything else, build in the engine and roll in

Reply:
Right now as long as libtessellator.dyllib is in a place that the process has in its `LD_SEARCH_PATH` it should "just work". 

Added a pubspec, filed https://github.com/flutter/flutter/issues/99563